fetch/main.js

```diff
@@ -0,0 +1,13 @@
+fetch('https://jsonplaceholder.typicode.com/users', {
+  method: 'GET'
+})
+  .then(response => response.json()
+    .then(data => console.log('typicode data:', data))
+  );
```

Comment:
This has the same end result but we typically chain the 2nd `.then` to the first `.then` instead of directly onto the `json` method:
```JAVASCRIPT
fetch('https://jsonplaceholder.typicode.com/users', {
  method: 'GET'
})
  .then(response => response.json())
  .then(data => console.log('typicode data:', data));
```

Reply:
ya that's what i saw in the example and tried it but it wasn't understanding the second .then was for the first one so idk i switched it to what i have now and it worked, i didn't really understand it